daemon/api_users.go

```diff
@@ -298,6 +269,7 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 }
 
 func createUser(c *Command, createData postUserCreateData) Response {
```

Comment:
we lost coverage of parts of this when compared to master, the path where enabled is set to true seems not reached anymore

Reply:
I restored some tests which covers this - I still need to verify we are back up on coverage, where did you see this?

Reply:
I just used the coverage tools locally:

go test -coverprofile=cov.out
go tool cover -html=cov.out

Reply:
OK so we are looking a lot better now it looks like. 94,5% here now vs 89,1% on master for api_users.go. The increased coverage is probably related to the reduction in code as well.